src/test/java/TaskListTest.java

```diff
@@ -0,0 +1,49 @@
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import java.util.ArrayList;
+
+public class TaskListTest {
+
+    @Test
+    public void getTaskListTest() {
```

Comment:
Perhaps you can separate each of the add task test into different tests e.g. for todo, for deadline and for event